.github/workflows/build.yaml

```diff
@@ -16,6 +16,8 @@ name: Builds
 
 on:
     push:
+        branches:
+            - 'master'
```

Comment:
Is this basically the same thing as https://github.com/project-chip/connectedhomeip/pull/32834 but for a different set of YAMLs?  If so, see my question there.

Reply:
Yes, Vatsal and I split the task between the two of us. I suspected there may be issues with this implementation so I had written up a Plan B in #30754; it would block CI only on "dependabot/**" branches. I believe that implementation should prevent the issues you mentioned.